plugins/woocommerce/src/Internal/Admin/Reviews.php

```diff
@@ -0,0 +1,603 @@
+<?php
+/**
+ * Products > Reviews
+ */
+
+namespace Automattic\WooCommerce\Internal\Admin;
+
+use WP_Ajax_Response;
+use WP_Comment;
+use WP_Screen;
+
+/**
+ * Handles backend logic for the Reviews component.
+ */
+class Reviews {
+
+	/**
+	 * Admin page identifier.
+	 */
+	const MENU_SLUG = 'product-reviews';
+
+	/**
+	 * Class instance.
+	 *
+	 * @var Reviews|null instance
+	 */
+	protected static $instance;
+
+	/**
+	 * Reviews page hook name.
+	 *
+	 * @var string|null
+	 */
+	protected $reviews_page_hook = null;
+
+	/**
+	 * Reviews list table instance.
+	 *
+	 * @var ReviewsListTable|null
+	 */
+	protected $reviews_list_table;
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+
+		add_action( 'admin_menu', [ $this, 'add_reviews_page' ] );
+		add_action( 'admin_enqueue_scripts', [ $this, 'load_javascript' ] );
+
+		// These ajax callbacks need a low priority to ensure they run before their WordPress core counterparts.
+		add_action( 'wp_ajax_edit-comment', [ $this, 'handle_edit_review' ], -1 );
+		add_action( 'wp_ajax_replyto-comment', [ $this, 'handle_reply_to_review' ], -1 );
+
+		add_filter( 'parent_file', [ $this, 'edit_review_parent_file' ] );
+
+		add_filter( 'gettext', [ $this, 'edit_comments_screen_text' ], 10, 2 );
+
+		add_action( 'admin_notices', [ $this, 'display_notices' ] );
+	}
+
+	/**
+	 * Gets the class instance.
+	 *
+	 * @return Reviews instance
+	 */
+	public static function get_instance() {
+
+		if ( null === self::$instance ) {
+			self::$instance = new self();
+		}
+
+		return self::$instance;
+	}
+
+	/**
+	 * Gets the required capability to access the reviews page and manage product reviews.
+	 *
+	 * @param string $context The context for which the capability is needed.
```

Comment:
```suggestion
	 * @param string $context The context for which the capability is needed.  (e.g. `view` or `moderate`)
```

I find it helpful to list the options. I had to look up the usage to see what the other possibility was.